daemon/algod/api/server/v2/test/helpers.go

```diff
@@ -115,7 +115,23 @@ func (m mockNode) BroadcastSignedTxGroup(txgroup []transactions.SignedTxn) error
 }
 
 func (m mockNode) GetPendingTransaction(txID transactions.Txid) (res node.TxnWithStatus, found bool) {
-	res = node.TxnWithStatus{}
+	t := transactions.Transaction{
```

Comment:
I think the modifications to this file are no longer needed since you're testing with e2e tests instead?